gym-unity/gym_unity/tests/test_gym.py

```diff
@@ -183,3 +188,33 @@ def setup_mock_unityenvironment(mock_env, mock_spec, mock_result):
     mock_env.return_value.get_agent_groups.return_value = ["MockBrain"]
     mock_env.return_value.get_agent_group_spec.return_value = mock_spec
     mock_env.return_value.get_step_result.return_value = mock_result
+
+
+@pytest.mark.parametrize("mapper_cls", [AgentIdIndexMapper, AgentIdIndexMapperSlow])
```

Comment:
That's pretty cool.

Reply:
Do you mean `pytest.mark.parametrize`, or passing class types as parameters?

Reply:
both